### PR TITLE
Create code-learning.tsx

### PR DESCRIPTION
**Pull request by adaaaam**

### CODE REVIEW

**Comment by adaaaam** [comment]
@aileenrose finished with the copy edits but couldn't figure out how to fix the Yelp logo... also padding around the video seems to be different than the other product pages but don't know how to fix.

**Comment by adaaaam** [comment]
@adamfrankl2015 @sqs adding you to this PR since Lori wants to review this with you

**Comment by sqs** [comment]
Will review this afternoon

File: website/src/pages/product/code-learning.tsx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+                                It’s challenging to easily explore and understand the content and structure of new and
+                                unfamiliar codebases.
+                            </p>
+                        </IconItem>
+                    </div>
+                    <div className="col-md-4 mb-4">
+                        <IconItem color="red" icon={HelpIcon} className="py-3">
+                            <p className="text-sans-serif">
+                                New engineering hires don’t know where to look for specific code in large and
+                                distributed libraries and repositories.
```
Comment (sqs) [inline website/src/pages/product/code-learning.tsx:71]: What does "distributed" mean in this case?

File: website/src/pages/product/code-learning.tsx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            />
+            <ContentSection color="white" className="py-2">
+                <h2 className="display-4 mb-4 text-center" id="why">
+                    Learning code is essential for both new hires and those assigned to new codebases
+                </h2>
+                <div className="row">
+                    <div className="col-md-4 mb-4">
+                        <IconItem color="red" icon={HeadSnowflakeIcon} className="py-3">
+                            <p className="text-sans-serif">
+                                Without contextual code intelligence, it’s difficult to find answers to code questions –
```
Comment (sqs) [inline website/src/pages/product/code-learning.tsx:54]: should be an em-dash

File: website/src/pages/product/code-learning.tsx [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                                structure, to track down where code lives, and self-service during their investigations.
+                            </p>
+                            <footer className="blockquote-footer">Owen Kim, Senior Software Engineer, Convoy</footer>
+                        </blockquote>
+                    </div>
+                    <div className="col-lg-10 mt-2 py-4 text-center">
+                        <h5>
+                            Learn more about how{' '}
+                            <a href="/blog/convoy-improved-their-developer-on-boarding-with-sourcegraph">
+                                Convoy uses Sourcegraph to onboard new developers.
```
Comment (sqs) [inline website/src/pages/product/code-learning.tsx:93]: The period should go outside of the link, because the link is not the full sentence. https://about.sourcegraph.com/handbook/communication/style_guide#general

File: website/src/pages/product/code-learning.tsx [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+                        </h5>
+                    </div>
+                </div>
+            </ContentSection>
+            <ContentSection color="white" className="pb-2">
+                <h2 id="features" className="display-4 mb-5 text-center">
+                    Why learning code is faster with Sourcegraph
+                </h2>
+                <p className="text-center mb-5">
+                    Sourcegraph <a href="/universal-code-search">Universal Code Search</a> lets you explore across your
```
Comment (sqs) [inline website/src/pages/product/code-learning.tsx:104]: This is a very long sentence. It also is from our POV. Can we shorten it and rephrase it to be more like "With Sourcegraph, you can..." instead of "Sourcegraph lets you"?

File: website/src/pages/product/code-learning.tsx [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+                    code and public codebases with contextual code intelligence to accelerate understanding code when
+                    onboarding new developers and reviewing unfamiliar code.
+                </p>
+                <div className="row">
+                    <div className="col-lg-6 mb-4">
+                        <IconItem className="position-relative" icon={GlobeIcon} color="green">
+                            <h4>Find definitions and explore references, even across projects</h4>
+                            <p>
+                                See a call to a function you don't recognize? With Sourcegraph, you can see its
+                                documentation and type signature by hovering over it &mdash; and go to its definition in
```
Comment (sqs) [inline website/src/pages/product/code-learning.tsx:114]: don't put spaces around em-dashes

File: website/src/pages/product/code-learning.tsx [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+                                a single click, across package, dependency, and repository boundaries. Even if your IDE
+                                can do this, Sourcegraph lets you explore code intelligently in your web browser in any
+                                repository on any branch, instantly and without losing your local context to massively
```
Comment (sqs) [inline website/src/pages/product/code-learning.tsx:117]: "to massively save time" sounds awkward coming at the end. It is also hard to parse - because "losing your local context to NOUN" is a common phrase, but this "to massively save time" applies to "explore code".

File: website/src/pages/product/code-learning.tsx [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+                    </div>
+                    <div className="col-lg-6 mb-4">
+                        <IconItem className="position-relative" icon={PlaylistAddCheckIcon} color="brand-cyan">
+                            <h4>
+                                Facilitate easier onboarding, with rapidly growing and distributed teams and codebases
+                            </h4>
+                            <p>
+                                New hires often spend the majority of their time jumping around unfamiliar parts of the
+                                codebase to build a mental model of the organization’s code. Sourcegraph accelerates
+                                this learning path with{' '}
```
Comment (sqs) [inline website/src/pages/product/code-learning.tsx:130]: "learning path" is an awkward phrase. I've never heard that term before, but as written, it sounds like it's a known term.

File: website/src/pages/product/code-learning.tsx [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+                                can do this, Sourcegraph lets you explore code intelligently in your web browser in any
+                                repository on any branch, instantly and without losing your local context to massively
+                                save time. Works for <SupportedProgrammingLanguagesLink />.
+                            </p>
+                        </IconItem>
+                    </div>
+                    <div className="col-lg-6 mb-4">
+                        <IconItem className="position-relative" icon={PlaylistAddCheckIcon} color="brand-cyan">
+                            <h4>
+                                Facilitate easier onboarding, with rapidly growing and distributed teams and codebases
```
Comment (sqs) [inline website/src/pages/product/code-learning.tsx:125]: "Facilitate" is a throwaway word. Can we just remove it? "Onboard faster"?

"Distributed teams" is mentioned, but we don't explain how that applies. The body text below explains how Sourcegraph helps you with big codebases, but it doesn't explain how it helps when your team is distributed. We could say something about how when you can't tap people on the shoulder, you need a way to find answers on your own. But that's jamming too many ideas into a single paragraph IMO.

Also "distributed" adjective seems to apply to "codebases", which does not make sense.

File: website/src/pages/product/code-learning.tsx [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+                                this learning path with{' '}
+                                <a href="https://about.sourcegraph.com/universal-code-search">universal code search</a>{' '}
+                                across programming languages, code hosts, repositories, version control systems,
+                                services, and APIs.
+                            </p>
+                        </IconItem>
+                    </div>
+                    <div className="col-lg-6 mb-4">
+                        <IconItem className="position-relative" icon={StopwatchIcon} color="brand-purple">
+                            <h4>Reduce the time to first commit</h4>
```
Comment (sqs) [inline website/src/pages/product/code-learning.tsx:139]: I would render this as "time-to-first-commit" to make this easier to parse when reaidng.

File: website/src/pages/product/code-learning.tsx [hunk 10]
```diff
@@ -0,0 +1,212 @@
...
+                    <div className="col-lg-6 mb-4">
+                        <IconItem className="position-relative" icon={PlaylistAddCheckIcon} color="brand-cyan">
+                            <h4>
+                                Facilitate easier onboarding, with rapidly growing and distributed teams and codebases
+                            </h4>
+                            <p>
+                                New hires often spend the majority of their time jumping around unfamiliar parts of the
+                                codebase to build a mental model of the organization’s code. Sourcegraph accelerates
+                                this learning path with{' '}
+                                <a href="https://about.sourcegraph.com/universal-code-search">universal code search</a>{' '}
```
Comment (sqs) [inline website/src/pages/product/code-learning.tsx:131]: UCS is capitalized elsewhere. My strong preference is to not capitalize it (as it's done here).

**Comment by aileenrose** [comment]
This page is no longer required.  Closing PR.
